less/components/footer.less

```diff
@@ -10,159 +10,130 @@
  */
 
 footer {
-	font-family: @font-family-secondary;
-}
+ font-family: @font-family-secondary;
+ color: @contrast-colour;
 
-footer p,
-footer li,
-footer a {
-	font-size: 16px;
-	font-family: @font-family-secondary;
-}
+ p, li, a, .title {
+	 font-size: 0.85rem;
+	 font-family: @font-family-secondary;
+ }
 
-footer ul {
-	padding-left: 0;
-	list-style-type: none;
-}
+ ul {
+	 padding-left: 0;
+	 list-style-type: none;
+ }
 
-footer a {
-	color: inherit;
+ a {
+	 color: inherit;
+
+	 &:hover, &:focus, &:active {
+		 color: @secondary-colour;
+	 }
+ }
 
-	&:hover {
-		color: #FFF;
-	}
 }
 
 .sidebar_wrapper {
-	display: flex;
-	flex-wrap: wrap;
+ display: flex;
+ flex-wrap: wrap;
 }
 
 @media (min-width: 992px) {
-	.container-footer {
-		.container-padding();
-	}
+ .container-footer {
+	 .container-padding();
+ }
 }
 
 .pkp_block {
-	-webkit-box-flex: 0;
-	position: relative;
-	padding-right: @site-padding;
-	padding-top: @site-padding/3;
-	padding-bottom: @site-padding/3;
+ -webkit-box-flex: 0;
+ position: relative;
+ padding-right: @site-padding;
+ padding-top: @site-padding/3;
+ padding-bottom: @site-padding/3;
+
+ .title {
+	 font-weight: bold;
+ }
 }
 
 @media (min-width: 991px) {
-	.pkp_block {
-		flex: 0 0 100%/4;
-	}
+ .pkp_block {
+	 flex: 0 0 100%/4;
+ }
 }
 
 @media (min-width: 768px) and (max-width: 990px) {
-	.pkp_block {
-		flex: 0 0 100%/2;
-		max-width: 100%/2;
-	}
+ .pkp_block {
+	 flex: 0 0 100%/2;
+	 max-width: 100%/2;
+ }
 }
 
 @media (max-width: 767px) {
-	.pkp_block {
-		flex: 0 0 100%;
-		max-width: 100%;
-	}
-}
-
-.pkp_block .title {
-	font-weight: bolder;
-}
-
-.pkp_block img {
-	width: 100%;
-	border-radius: 5px;
-	padding: 1px;
-	border: 1px solid #ddd;
-}
-
-.pkp_block.block_web_feed img {
-	width: auto;
-	border-radius: 0;
-	padding: 0;
-	border: 0;
-}
-
-.block_web_feed ul {
-	list-style-type: none;
-	padding: 0;
-}
-
-.block_web_feed {
-	text-align: left;
+ .pkp_block {
+	 flex: 0 0 100%;
+	 max-width: 100%;
+ }
 }
 
 .block_make_submission_link {
-	.button();
-	border: 1px solid #000;
-	background-color: #000;
-	color: white;
-	display: inline-block;
-	text-align: center;
-	font-family: @font-family-secondary;
-	line-height: 24px;
-	font-size: 18px;
-
-	&:hover {
-		text-decoration: none;
-		background-color: #FFF;
-		border: 1px solid #FFF;
-		color: #000;
+	&:extend(.btn);
+	border-color: @secondary-colour;
+	background-color: @secondary-colour;
+	color: contrast(@contrast-colour);
+
+	&:hover, &focus, &:active {
```

Comment:
`&focus` here is missing the `:`

Reply:
resolved in a commit that’s been force-pushed... But it looks like the Github PR UI is [having some delays](https://www.githubstatus.com/)